docs/jwst/outlier_detection/outlier_detection_coron.rst

```diff
@@ -25,9 +24,6 @@ Specifically, this routine performs the following operations:
 #. Perform statistical comparison between median image and original image to identify outliers.
 
    The core detection algorithm uses the following to generate an outlier mask
-
```

Comment:
I think this blank line is needed - the equation below doesn't render without it.